packages/core/src/lib/callback-runner.ts

```diff
@@ -1,12 +1,19 @@
-const some = require('./async-every')
+import some from './async-every'
 
-module.exports = (callbacks, event, onCallbackError, cb) => {
+type NodeCallbackType<T = any> = (error?: Error | null, result?: T) => void;
+
+const runCallbacks = <T>(
+  callbacks: any[],
```

Comment:
Wanted to specify the type here based on usage of this function. 
```suggestion
  callbacks: (OnErrorCallback | undefined)[],
```
But got errors from callback-runner.test as we are using other types there. I'm assuming that we expecting any type of `callbacks`.